Lib/tkinter/__init__.py

```diff
@@ -590,9 +590,20 @@ def get(self):
             raise ValueError("invalid literal for getboolean()")
 
 
-def mainloop(n=0):
-    """Run the main loop of Tcl."""
-    _default_root.tk.mainloop(n)
+def mainloop(threshold=0):
```

Comment:
This is an API change that should be flagged?

Reply:
Good point, this was just meant to make the variable name a bit more descriptive, but it could break (rare) user code that relied on setting `n` as a keyword argument.